Bitbucket-Pipelines/two-stage-pipeline-template/questions.json

```diff
@@ -0,0 +1,174 @@
+{
+  "questions": [{
+    "key": "intro",
+    "question": "\nThis template configures a pipeline that deploys a serverless application to a testing and a production stage.\n",
+    "kind": "info"
+  }, {
+    "key": "pipeline_user_aws_access_key_id_variable_name",
+    "question": "What is the BitBucket variable key for pipeline user account access key ID?",
+    "default": "AWS_ACCESS_KEY_ID"
+  }, {
+    "key": "pipeline_user_aws_secret_access_key_variable_name",
+    "question": "What is the BitBucket variable key for pipeline user account access key secret?",
+    "default": "AWS_SECRET_ACCESS_KEY"
+  }, {
+    "key": "main_git_branch",
+    "question": "What is the git branch used for production deployments?",
+    "default": "main"
+  }, {
+    "key": "sam_template",
+    "question": "What is the template file path?",
+    "default": "template.yaml"
+  }, {
+    "key": "message_testing_stage_name",
+    "question": "We use the stage name to automatically retrieve the bootstrapped resources created when you ran `sam pipeline bootstrap`.\n",
```

Comment:
Simon has a PR to change all "stage name" to "configuration name," do we want to do the same here or wait

Reply:
Good spot I've updated them to be consistent